modules/import-export/mod-pcm/ImportPCM.cpp

```diff
@@ -353,27 +353,30 @@ void PCMImportFileHandle::Import(
 
          if (block) {
             unsigned c = 0;
-            ImportUtils::ForEachChannel(*track, [&](auto& channel)
```

Comment:
There is a ImportUtils::ForEachChannel(TrackList&, const std::function<void(WaveChannel&)>&) overload